revscoring/languages/spanish.py

```diff
@@ -135,15 +134,16 @@
     r"marimach[ao]",  # tomboy
     r"maripos[óo]n",  # fag
     r"marrano",  # pig
-    r"meachupan", r"meapelan",  # something 'suck me' ???
+    r"meachupan", r"meapelan",  # suck my cock
     r"me[- ]?castra[- ]?la[- ]?madre",  # 'I <something> your mother' ???
-    r"me[- ]?la[- ]?pelan?s?",  # "You ain't shit" ???
+    r"me[- ]?la[- ]?pelan?s?",  # I don't give a f***
     r"mi?er[dg]a+s?\w*",  # shit
     r"mimsn",  # ???
     r"ming[ao]",  # ugly person
     r"mocos",  # boogers / mucus
     r"mojon",  # shit
     r"monda",  # man part / penis
+    r"mongolo", # idiot (slang)
```

Comment:
flake8 is failing here.  There should be two spaces before the comment "#"

Reply:
Fixing.